src/rust/bitbox02-rust-c/src/lib.rs

```diff
@@ -46,5 +46,6 @@ fn panic(info: &core::panic::PanicInfo) -> ! {
     ::util::log::log!("{}", info);
     #[cfg(feature = "firmware")]
     bitbox02_rust::print_screen!(0, "Error: {}", info);
+    cortex_m::asm::bkpt();
```

Comment:
What's this about? Maybe should be a separate commit/PR with a commit msg.

Reply:
Put it in a separate commit. It is a breakpoint so exeuction stops when there is a panic.